app/theme/brave_theme_resources.grd

```diff
@@ -22,6 +22,7 @@
       <structure type="chrome_scaled_image" name="IDR_PRODUCT_LOGO_32_DEV" file="brave/product_logo_32_dev.png" />
       <structure type="chrome_scaled_image" name="IDR_PRODUCT_LOGO_32_DEVELOPMENT" file="brave/product_logo_32_development.png" />
       <if expr="not is_android">
+        <structure type="chrome_scaled_image" name="IDR_TOR_WINDOW_FRAME_GRAPHIC" file="brave/tor_window_frame_graphic.png" />
```

Comment:
Out of interest - did you try a scalable .icon instead of PNG?

Reply:
I didn't try because .icon only works well for icon type(square) and mono color.